src/Nethermind/Nethermind.Evm/CodeAnalysis/JumpDestinationAnalyzer.cs

```diff
@@ -98,25 +103,167 @@ private void AnalyzeJumpDestinations(out object previous)
     private static int GetInt64ArrayLengthFromBitLength(int n) =>
         (n - 1 + (1 << BitShiftPerInt64)) >>> BitShiftPerInt64;
 
-    /// <summary>
-    /// Collects data locations in code.
-    /// An unset bit means the byte is an opcode, a set bit means it's data.
-    /// </summary>
     private long[] CreateJumpDestinationBitmap()
     {
         Metrics.IncrementContractsAnalysed();
         ReadOnlySpan<byte> code = MachineCode.Span;
-        long[] jumpDestinationBitmap = new long[GetInt64ArrayLengthFromBitLength(code.Length)];
+
+        long[] bitmap = CreateBitmap(code.Length);
+
+        return Vector512<sbyte>.IsSupported && code.Length >= Vector512<sbyte>.Count ?
+            CreateJumpDestinationBitmap_Vector512(bitmap, code) :
+            Vector128<sbyte>.IsSupported && code.Length >= Vector128<sbyte>.Count ?
+            CreateJumpDestinationBitmap_Vector128(bitmap, code) :
+            CreateJumpDestinationBitmap_Scalar(bitmap, code);
+    }
+
+    internal static long[] CreateBitmap(int codeLength)
+        => new long[GetInt64ArrayLengthFromBitLength(codeLength)];
+
+    [SkipLocalsInit]
+    internal static long[] CreateJumpDestinationBitmap_Scalar(long[] bitmap, ReadOnlySpan<byte> code)
```

Comment:
it has `Create` but doesn't create anything?

Reply:
Rename